test/examples/examples.js

```diff
@@ -67,21 +73,22 @@ const cases = [
   }, 'Vote nay'],
   [{
     source: 'Token `_amount / 10^18`',
-    bindings: { _amount: int(new BigNumber(10).times(Math.pow(10, 18))) }
+    bindings: { _amount: int(new BN(10).mul(new BN(10).pow(new BN(18)))) }
   }, 'Token 10'],
   [{
     source: '`_bool ? \'h\' + _var + \'o\' : \'damn\'`',
     bindings: { _bool: bool(true), _var: string('ell') }
   }, 'hello']
 ]
 
-test('Examples', async (t) => {
-  for (let [input, expected] of cases) {
+for (let [input, expected] of cases) {
+  test(input.source, async (t) => {
+    t.pass()
```

Comment:
What's this for?

Reply:
Oh crap, good catch! I was fiddling with this test when I was making the changes and used `t.pass()` to just skip this suite 😅